src/app/home-page/home-logged-in/home-logged-in.component.html

```diff
@@ -64,9 +64,9 @@ <h3>Recent Activity</h3>
         <div class="w-100">
           <h3>Links</h3>
           <ul>
-            <li><a routerLink="quick-start">Quickstart</a></li>
-            <li><a routerLink="accounts">My Account</a></li>
-            <li><a routerLink="starred">My Starred</a></li>
+            <li><a [routerLink]="['../quick-start']">Quickstart</a></li>
```

Comment:
I think it would be better to just have, for example, `['/quick-start']`, without the `..`; that way if this component gets moved to a different location (unlikely, but still), the links will continue to work.

On the other hand, you could argue that the links will always be siblings of this page, but I feel it's more flexible to remove the `..`.

Reply:
oh yup I agree :+1: better to be more flexible